travelrec-web/src/app/app.component.ts

```diff
@@ -15,20 +15,33 @@ export class AppComponent {
   @ViewChild('lgModal') lgModal: any;
 
   recommendations = [];
+  errorMessage = null;
   detailsList = [];
   modalName = '';
   query: string;
 
+  errors = {
+    1: "Sorry, we didn't find any location in your query.",
```

Comment:
Can't you add a field to error with an appropriate message instead of this dictionaries?

Reply:
done